core/services/relay/evm/evm.go

```diff
@@ -75,18 +76,39 @@ func (r *Relayer) NewConfigProvider(args relaytypes.RelayArgs) (relaytypes.Confi
 
 type configWatcher struct {
 	utils.StartStopOnce
+	lggr             logger.Logger
 	contractAddress  common.Address
 	contractABI      abi.ABI
 	offchainDigester types.OffchainConfigDigester
 	configPoller     *ConfigPoller
 	chain            evm.Chain
+	new              bool
```

Comment:
nit. rename to runReplay? 'new' is a job concept and it can decide whether to run replay or not, the configWatcher needn't be aware of job semantics